protocol/messenger.go

```diff
@@ -2035,6 +2035,8 @@ func (m *Messenger) sendChatMessage(ctx context.Context, message *common.Message
 		audioMessage.Payload = payload
 		audioMessage.Type = audio.Type(payload)
 		message.Payload = &protobuf.ChatMessage_Audio{Audio: audioMessage}
+
+		_ = file.Close()
```

Comment:
this is bug fix, not directly related to the new feature, since this code is unused.

Reply:
Perhaps you can defer the file close so that the file will be guaranteed to be closed when the func ends
```suggestion
		defer file.Close()
```

Reply:
it is deferred at the top of the scope.
the problem is that below it tries to delete this file, and since defer is not executed yet, the file remains locked and this results in error.

Reply:
Ah I see it. I didn't see the top of the scope.